lib/core/presentation/pages/collaborator/sub_pages/widgets/collaborator_filter_bottomsheet/collaborator_filter_bottomsheet.dart

```diff
@@ -0,0 +1,141 @@
+import 'package:app/core/presentation/pages/collaborator/sub_pages/widgets/collaborator_user_bottomsheet_header/collaborator_user_bottomsheet_header.dart';
+import 'package:app/core/presentation/widgets/bottomsheet_grabber/bottomsheet_grabber.dart';
+import 'package:app/core/presentation/widgets/common/button/linear_gradient_button_widget.dart';
+import 'package:app/core/presentation/widgets/lemon_text_field.dart';
+import 'package:app/core/presentation/widgets/theme_svg_icon_widget.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/theme/sizing.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:flutter/material.dart';
+
+final mockData = [
+  "Create new concept",
+  "Design advice",
+  "Perform video editing",
+  "Financial advice",
+  "Execute email marketing",
+  "Manage budget",
+  "Develop marketing strategy",
+  "Write blog posts",
+  "Optimize website performance",
+  "Provide customer support",
+  "Conduct market research",
+];
+
+class CollaboratorFilterBottomSheet extends StatelessWidget {
+  const CollaboratorFilterBottomSheet({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final t = Translations.of(context);
+    final colorScheme = Theme.of(context).colorScheme;
+    return Container(
+      padding:
+          EdgeInsets.only(bottom: MediaQuery.of(context).viewInsets.bottom),
```

Comment:
did you try to focus on the keyboard ? when the keyboard appear, the height of bottom sheet will be shorter => overflowed error
to avoid this , wrap component with SingleChildScrollView